config/develop/example-project.yaml

```diff
@@ -0,0 +1,26 @@
+template_path: tower-project.yaml
+stack_name: example-project
+dependencies:
+  - develop/nextflow-vpc.yaml
+  - common/nextflow-forge-iam-policy.yaml
+  - common/nextflow-launch-iam-policy.yaml
+
+parameters:
+  ReadWriteAccessArns:
+    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/bruno.grande@sagebase.org
+  ReadOnlyAccessArns:
+    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/tess.thyer@sagebase.org
+  AdminArns:
+    - {{stack_group_config.sso_admin_role.arn}}
```

Comment:
i guess you can also do a lookup with `!rcmd` resolver if you want, although i'm not sure that's much better than passing in static string.. 

```
- !rcmd  aws iam get-role --role-name AWSReservedSSO_Administrator_5b839ce6e3a82407 --query 'Role.Arn'
```

Reply:
In this case, this adds a command that can be skipped. That said, if ever we need any other information on the role, I like this approach since you gain access to everything returned by `get-role`.